docs/modify-cluster.md

```diff
@@ -0,0 +1,24 @@
+# Modifying your cluster
+
+You can do the following cluster administrative tasks.
+
+## Prerequisites
+
+1. You created your [account roles using Terraform](../examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md).
+1. You created your cluster using Terraform. This cluster can either have [a managed OIDC configuration](../examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md) or [an unmanaged OIDC configuration](../examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_unmanaged_oidc_config/README.md).
+
+## Common machine pool  tasks
+
+You may need to perform any of the following administrative tasks on your cluster:
```

Comment:
It's not clear if those tasks will be done on default or non-default machine pool ...
Also, I think need to point on examples